io/disk/multipath_test.py

```diff
@@ -118,12 +120,64 @@ def test(self):
                 if not multipath.device_exists(path_dic["wwid"]):
                     msg += "Recovery of %s fails\n" % path_dic["wwid"]
 
-            # Blacklist sdX
-            for disk in path_dic["paths"]:
-                cmd = "devnode %s" % disk
-                multipath.form_conf_mpath_file(blacklist=cmd)
-                if disk in multipath.get_paths(path_dic["wwid"]):
-                    msg += "Blacklist of %s fails\n" % disk
+            # Failing and reinstating individual paths eg: sdX
```

Comment:
I would keep the previous method of failing and reinstating paths also in the test, since they are 2 different ways of doing the same and no harm in having it.